syntax = "proto3";

message Airline {
    int32 id = 1;
    string title = 2;
}

message AirlinesRankingByTicketPriceRequest {
    
}

message AirlinesRankingByTicketPriceResponse {
    repeated Airline airline = 1;
}

service Ranking {
    rpc GetAirlinesRankingByTicketPrice (AirlinesRankingByTicketPriceRequest) returns (AirlinesRankingByTicketPriceResponse);
}
